d3b_dff_cli/modules/validation/check_manifest.py

```diff
@@ -43,14 +45,30 @@ def validate_row(row, rules):
                 if col == "file_size" and row.get("file_format", "").lower() in ["fastq", "bam", "cram"]:
                     greater_than_value = consequence.get("greater_than")
                     if greater_than_value:
-                        try:
-                            file_size_in_gb = float(row.get(col, 0)) / (1024 * 1024 * 1024)  # Convert to GB
-                            if file_size_in_gb <= float(greater_than_value.rstrip(" GB")):
-                                error_messages.append(f"Warning: *{col}* less than {greater_than_value}")
-                        except ValueError:
-                            error_messages.append(f"*{col}* is not a valid numeric value")
+                        experiment = row.get("experiment_strategy", "").lower()
+                        if experiment in ["wgs", "wxs", "wes"]:
+                            greater_than_value = "1 GB"
```

Comment:
Is it worth thinking about standardizing how we assign these variables? We have some in the json file and are hard coding others here. It might simplify the code to store variables specific to the files types and cut down on the if statements.

Reply:
> Is it worth thinking about standardizing how we assign these variables? We have some in the json file and are hard coding others here. It might simplify the code to store variables specific to the files types and cut down on the if statements.

We set the standard cutoff to 200M in the json file, as this value may change when new sequencing experiments are included (e.g., single cell). But, for WGS/WXS, we’re confident that the file size must be greater than 1 GB, so this value is hardcoded.

Reply:
Yeah, I was suggesting should we add in variables for different greater_than_values, i.e. greater_than_value_wgs_wxs_wes so that we are managing variables assignments in one place.